dashboard/app/models/level.rb

```diff
@@ -35,6 +36,7 @@ class Level < ActiveRecord::Base
   has_many :hint_view_requests
 
   before_validation :strip_name
+  before_save :log_changes
```

Comment:
ah, very good point. I'll dig a little deeper to see if I can distinguish between the two save cases.